chains/solana/contracts/programs/ccip-router/src/lib.rs

```diff
@@ -392,18 +293,11 @@ pub mod ccip_router {
         mint: Pubkey, // should we validate that this is a real token program?
         token_admin_registry_admin: Pubkey,
     ) -> Result<()> {
-        let token_admin_registry = &mut ctx.accounts.token_admin_registry;
-        token_admin_registry.version = 1;
-        token_admin_registry.administrator = token_admin_registry_admin;
-        token_admin_registry.pending_administrator = Pubkey::new_from_array([0; 32]);
-        token_admin_registry.lookup_table = Pubkey::new_from_array([0; 32]);
-
-        emit!(AdministratorRegistered {
-            token_mint: mint,
-            administrator: token_admin_registry_admin,
-        });
-
-        Ok(())
+        v1::tokens::register_token_admin_registry_via_get_ccip_admin(
```

Comment:
nit: rename tokens to token-admin-registry? And then fee_quoter could be a part of it as well

Reply:
I'd rather keep fee_quoter separate from the rest of the tokens logic, as the fee calculation will have a lot of code unrelated to general token management. I'll make the rename though :+1: